autoscale/src/main/java/com/sequenceiq/periscope/service/AlertService.java

```diff
@@ -266,6 +267,11 @@ public void addPrometheusAlertsToConsul(Cluster cluster) {
         return prometheusAlertRepository.findAllByCluster(clusterId);
     }
 
+    private Cluster getClusterById(Long clusterId) {
+        Optional<Cluster> clusterOpt = clusterRepository.findById(clusterId);
+        return clusterOpt.orElseThrow(() -> new NotFoundException(String.format("Cluster hasn't found with id: '%s'", clusterId)));
```

Comment:
hasn't found sounds strange to me. can we change it to `is not found` or `has not been found`?